js/sr_tools_db.js

```diff
@@ -1722,91 +1731,629 @@ var db = {
+				res.max_attributes.logic += -1;
+				res.attributes.intuition += -1;
+				res.max_attributes.intuition += -1;
+				res.attributes.charisma += -2;
+				res.max_attributes.charisma += -2;
+				res.attributes.magic += 6;
+				break;
+
+			case 'Bandersnatch':
+				res.attributes.body += 5;
+				res.min_attributes.body += 5;
+				res.max_attributes.body += 5;
+				res.attributes.strength += 4;
+				res.min_attributes.strength += 4;
+				res.max_attributes.strength += 4;
+				res.attributes.magic += 6;
+				break;
+			case 'Dzoo-Noo-Qua':
+				res.attributes.body += 4;
+				res.min_attributes.body += 4;
+				res.max_attributes.body += 4;
+				res.attributes.agility += -1;
+				res.max_attributes.agility += -1;
+				res.attributes.strength += 4;
+				res.min_attributes.strength += 4;
+				res.max_attributes.strength += 4;
+				res.attributes.logic += -1;
+				res.max_attributes.logic += -1;
+				res.attributes.intuition += -1;
+				res.max_attributes.intuition += -1;
+				res.attributes.charisma += -2;
+				res.max_attributes.charisma += -2;
+				res.attributes.magic += 6;
+				break;
+
+			case 'Centaur':
+				res.attributes.body += 2;
+				res.min_attributes.body += 2;
+				res.max_attributes.body += 2;
+				res.attributes.strength += 2;
+				res.min_attributes.strength += 2;
+				res.max_attributes.strength += 2;
+				res.attributes.intuition += -1;
+				res.max_attributes.intuition += -1;
+				res.attributes.charisma += -1;
+				res.max_attributes.charisma += -1;
+				res.attributes.edge += -1;
+				res.max_attributes.edge += -1;
+				res.attributes.magic += 1;
+				res.augmentations.push({name: 'Low-Light Vision'});
+				res.augmentations.push({name: 'Thermographic Vision'});
+				res.augmentations.push({name: 'Magic Sense'});
+				res.augmentations.push({name: 'Natural Weapon (Kick: DV(STR+2)P, AP +1, Reach +1)'});
+				res.augmentations.push({name: 'Search'});
+				res.augmentations.push({name: 'Movement (x1/x4/+4)'});
+				break;
+
+			case 'Naga':
+				res.attributes.body += 2;
+				res.min_attributes.body += 2;
+				res.max_attributes.body += 2;
+				res.attributes.agility += -2;
+				res.max_attributes.agility += -2;
+				res.attributes.reaction += 1;
+				res.min_attributes.reaction += 1;
+				res.max_attributes.reaction += 1;
+				res.attributes.strength += 3;
+				res.min_attributes.strength += 3;
+				res.max_attributes.strength += 3;
+				res.attributes.will += 1;
+				res.min_attributes.will += 1;
+				res.max_attributes.will += 1;
+				res.attributes.charisma += 1;
+				res.min_attributes.charisma += 1;
+				res.max_attributes.charisma += 1;
+				res.attributes.edge += -1;
+				res.max_attributes.edge += -1;
+				res.attributes.magic += 1;
+				res.augmentations.push({name: 'Armor (8)'});
+				res.augmentations.push({name: 'Cold-Blooded'});
+				res.augmentations.push({name: 'Dual Natured'});
+				res.augmentations.push({name: 'Guard'});
+				res.augmentations.push({name: 'Natural Weapon (Bite: DV(STR+1)P, AP -2, Reach -1)'});
```

Comment:
The same comment about the centaur's kick applies to the naga's bite.

Reply:
Same as with the other natural weapons